source/braille.py

```diff
@@ -1162,8 +1162,7 @@ def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):
 				textList.append(_("h%s") % headingLevel)
 		collapsed = field.get("collapsed")
 		if collapsed:
```

Comment:
Why does this need to be assigned to a variable?
```suggestion
		if field.get("collapsed"):
```